src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,21 @@
+package duke.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDate from;
+    protected LocalDate to;
```

Comment:
Consider making this private?
```suggestion
    private LocalDate from;
    private LocalDate to;
```